src/layouts/components/Header.tsx

```diff
@@ -132,18 +145,42 @@ export function Header(): JSX.Element {
       {menu && (
         <>
           <div className="fixed z-50 md:hidden">
-            <MobileMenu toggleMenu={() => setMenu(false)} />
+            <MobileMenu
+              toggleMenu={() => setMenu(false)}
+              openProposals={openProposals}
```

Comment:
instead of passing an optional prop that will not be used by all menu items, let's modify the original `drawerMenuItemLinks`